### PR TITLE
A Better search

### PR DESCRIPTION
**Pull request by Deewai**
This adds a better search than #1 . users can search by either momentID or listingID
Screenshot 2020-11-22 at 11 43 54 AM

### CODE REVIEW

File: src/pages/account.comp.js [hunk 1]
```diff
@@ -200,6 +226,51 @@ export function Account() {
...
+  }
+  const handleSearchListing = ()=>{
+    setListingError(null)
+    if(searchListingID == null || searchListingID == ""){
+      setSaleMomentIDs(topshotAccount.saleMomentIDs.slice(0, 20))
+      return
+    }
+    let value = parseInt(searchListingID)
+    //search the list of saleMomentIDs
+    if(topshotAccount.saleMomentIDs.includes(value)){
```
Comment (rrrkren) [inline src/pages/account.comp.js:265]:
```suggestion
    if(!topshotAccount.saleMomentIDs.includes(value)){
```

File: src/pages/account.comp.js [hunk 2]
```diff
@@ -200,6 +226,51 @@ export function Account() {
...
+
+  const handleSearchMoment = ()=>{
+    setMomentError(null)
+    if(searchMomentID == null || searchMomentID == ""){
+      setMomentIDs(topshotAccount.momentIDs.slice(0, 20))
+      return
+    }
+    let value = parseInt(searchMomentID)
+    //search the list of momentIDs
+    if(topshotAccount.momentIDs.includes(value)){
```
Comment (rrrkren) [inline src/pages/account.comp.js:250]:
```suggestion
    if(!topshotAccount.momentIDs.includes(value)){
```

**Review by rrrkren — CHANGES_REQUESTED** [review]
I think the logic for momentID exist in collection is flipped?

File: src/pages/account.comp.js (hunk 1 above)
Comment (Deewai) [reply src/pages/account.comp.js]: oops, yea. Fixed

**Review by rrrkren — APPROVED** [review]

**Pull request merged**